lib/features/add_property/view/widgets/add_features_view.dart

```diff
@@ -0,0 +1,155 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:hatspace/data/property_data.dart';
+import 'package:hatspace/dimens/hs_dimens.dart';
+import 'package:hatspace/gen/assets.gen.dart';
+import 'package:hatspace/strings/l10n.dart';
+import 'package:hatspace/theme/hs_theme.dart';
+
+class AddFeaturesView extends StatefulWidget {
```

Comment:
```suggestion
class AddPropertyFeaturesView extends StatefulWidget {
```

Reply:
@hophongit98 I don't think we need to add Property word here, because the file is inside add_property feature.

Reply:
From my experience: the format: featureName_widgetName_suffix helps us get an overview when searching for a feature in the source code.
Ex: you only search: "AddProperty", you can find all the classes of this feature. In your case: AddFeaturesView won't be displayed when I search "AddProperty"
@suesitran how do you think?

Reply:
@hophongit98 Ok got your point, let's me update that